hwc2_device/HwcLayer.cpp

```diff
@@ -252,6 +254,26 @@ void HwcLayer::ImportFb() {
     return;
   }
 
+  layer_data_.bi->use_shadow_fds = (intel_i915_fd() >= 0);
```

Comment:
for dgpu passthrough case, will it also do such blit?

Reply:
Yes, this has nothing much to do with dGPU SR-IOV or dGPU pass-through. As long as rendering with dGPU and displaying with iGPU, we need this code path.